hack/make/.ensure-syscall-test

```diff
@@ -4,5 +4,20 @@ set -e
 # Build a C binary for cloning a userns for seccomp tests
 # and compile it for target daemon
 if [ "$DOCKER_ENGINE_GOOS" = "linux" ]; then
-	docker build ${DOCKER_BUILD_ARGS} -qt syscall-test contrib/syscall-test > /dev/null
+	if [ "$DOCKER_ENGINE_GOARCH" = "$DOCKER_CLIENT_OSARCH" ]; then
```

Comment:
`DOCKER_ENGINE_GOARCH` still should be `DOCKER_ENGINE_OSARCH` or this block will never run, right?